src/CommitmentsV2/SFA.DAS.CommitmentsV2/Mapping/CreateCohortRequestToAddCohortRequestMapper.cs

```diff
@@ -0,0 +1,29 @@
+﻿using SFA.DAS.CommitmentsV2.Api.Types;
+using SFA.DAS.CommitmentsV2.Api.Types.Requests;
+using SFA.DAS.CommitmentsV2.Application.Commands.AddCohort;
+
+namespace SFA.DAS.CommitmentsV2.Mapping
+{
+    public class CreateCohortRequestToAddCohortRequestMapper : IMapper<CreateCohortRequest, AddCohortCommand>
+    {
+        public AddCohortCommand Map(CreateCohortRequest source)
```

Comment:
Just out of interest. I'm assuming we are not planning on using AutoMapper within our project

Reply:
Personally I don't like AutoMapper.